packages/inter-protocol/src/proposals/replaceCommitteeAndCharter.js

```diff
@@ -337,12 +335,12 @@ const shutdown = async ({
   trace('EC Charter shutdown successful');
 };
 
-const main = async permittedPowers => {
-  await shutdown(permittedPowers);
+export const replaceCommitteeAndCharter = async permittedPowers => {
+  // await shutdown(permittedPowers);
   await startNewEconomicCommittee(permittedPowers);
   await startNewEconCharter(permittedPowers);
   await addGovernorsToEconCharter(permittedPowers);
-  await handlehighPrioritySendersList(permittedPowers);
+  // await handlehighPrioritySendersList(permittedPowers);
 
```

Comment:
Commented line 338. Because the `shutdown` definition assumes that committee and charter contracts have `shutdown` method in their creatorFacet. 

Commented 343 too. Although it's not required if I've updated the wallet addresses on line 5 in `runConfig` object. Wallet addresses should be based on a3p.